examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/BiteBananaActivity.java

```diff
@@ -33,9 +35,9 @@ public ComputedAttributes run(final Mission mission) {
     final var newFlag = bigBiteSize ? Flag.B : Flag.A;
     mission.flag.set(newFlag);
     mission.fruit.subtract(biteSize);
-    return new ComputedAttributes(bigBiteSize, newFlag);
+    return new ComputedAttributes(bigBiteSize, List.of(newFlag));
   }
 
   @AutoValueMapper.Record
-  public record ComputedAttributes(boolean biteSizeWasBig, Flag newFlag) {}
+  public record ComputedAttributes(boolean biteSizeWasBig, List<Flag> newFlag) {}
```

Comment:
I don't understand the purpose of this change, it's always a one-element list.

Reply:
Good point - the purpose was to reproduce a bug where generic record components would cause an exception to be thrown during annotation processing.

I think the RecordParameter example should cover this case, so I'll go ahead and revert BiteBanana 👍

Reply:
I've reverted all changes to BiteBanana - they were useful when I was experimenting, but don't make sense in this activity